src/Telegram.Bot/TelegramBotClient.cs

```diff
@@ -797,12 +911,14 @@ public Task<User> GetMeAsync(CancellationToken cancellationToken = default)
             int replyToMessageId = default,
             IReplyMarkup replyMarkup = default,
             CancellationToken cancellationToken = default,
-            Emoji? emoji = default) =>
+            Emoji? emoji = default,
+            bool allowSendingWithoutReply = default) =>
```

Comment:
The default convention in .NET world is to setup `CancellationToken` at the end of the calling params.